wrangler-core/src/main/java/io/cdap/directives/parser/ParseAvro.java

```diff
@@ -77,7 +77,7 @@ public UsageDefinition define() {
     UsageDefinition.Builder builder = UsageDefinition.builder(NAME);
     builder.define("column", TokenType.COLUMN_NAME);
     builder.define("schema-id", TokenType.IDENTIFIER);
-    builder.define("encode-type", TokenType.IDENTIFIER);
+    builder.define("encoding-type", TokenType.IDENTIFIER);
```

Comment:
this is a backwards incompatible change and cannot be done. If somebody upgrades a pipeline that is using 'encode-type' it will now break.

Reply:
Is there a compatible way to make sure that there is no confusion regarding the encoding type?

Reply:
we can update documentation but we cannot make backwards incompatible API changes

Reply:
So I can change the javadoc and the README.md file. I think those are the only places where this info is documented.

Reply:
These are used as usage definition and are really not stored anywhere other than in Directive. So the change is fully compatible.